zuul.d/molecule.yaml

```diff
@@ -471,6 +471,7 @@
     - ^.config/molecule/.*
     - ^roles/dnsmasq/(defaults|files|handlers|library|lookup_plugins|module_utils|tasks|templates|vars).*
     - ^roles/networking_mapper/(defaults|files|handlers|library|lookup_plugins|module_utils|tasks|templates|vars).*
+    - ^roles/config_drive/(defaults|files|handlers|library|lookup_plugins|module_utils|molecule|tasks|templates|vars).*
```

Comment:
I think we should also add `name: cifmw-molecule-nat64_appliance` molecule job to run on config_drive role changes. ~ L510

Reply:
care to provide another PR to get it covered?

Reply:
yes, we can do it in a separate PR.